src/site/_includes/pattern-set.njk

```diff
@@ -92,16 +92,18 @@ pageScripts:
 
     {# Show patterns from current set (if set) or from the whole suite #}
     {% if currentPatternSet %}
-
-      <h2> {{ currentPatternSet.title }} </h2>
-      {{ currentPatternSet.content | safe }}
       {% for patternId, pattern in patterns.patterns %}
         {% if pattern.set == currentPatternSet.id %}
-          <div class="flow">
-            <h3 id="{{pattern.id}}"><a href="/patterns/{{pattern.id}}">{{pattern.title}}</a></h3>
-            {{ pattern.content | safe }}
+          {% Details %}
+            {% DetailsSummary %}
+            {{pattern.title}}
+            {% endDetailsSummary %}
             {% CodePattern pattern.id %}
-          </div>
+            <p>{{ pattern.description | safe }}
+            <br>
+            <a href="/patterns/{{pattern.id}}">Learn more</a>
```

Comment:
If there's no description there will just be a blank line before this. Can we use two paragraphs?

Alternatively, could this look like:

```html
{% Details %}
  {% DetailsSummary %}
    {{pattern.title}}
  {% endDetailsSummary %}
  {% CodePattern pattern.id %}
  {{ pattern.description | md | safe }}
  <p><a href="...">Learn more</a></p>
{% endDetails %}
```

... this would allow folks to write Markdown if they like, but if they provide an empty string it won't render anything. We do this in the code sections of DCC.

Reply:
Description is provided in the fromtmatter (usually max 1 sentence) so I don't think we need to render md here.